README.md

```diff
@@ -206,7 +206,7 @@ web3 contract call --amount AMOUNT --address CONTRACT_ADDRESS --abi CONTRACT_ABI
 **Parameters:**
 
 - CONTRACT_ADDRESS - the address of the deployed contract
-- CONTRACT_ABI_FILE - the abi file of the deployed contract
+- CONTRACT_ABI_FILE - the abi file of the deployed contract (take into account that there are some bundled abi files like erc20.abi and erc721.abi so you could use them without downloading or compiling them)
```

Comment:
Where are the actual abi files?

Reply:
oh sorry they are in folder which is not in the repo

Reply:
added abi folder with 2 abi contracts, so to add more contracts just put them into folder and you could use them from the folder and from the binary which is convenient